plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanManager.kt

```diff
@@ -322,14 +322,26 @@ class CodeWhispererCodeScanManager(val project: Project) {
         return errorMessage
     }
 
+    private fun getCodeScanExceptionMessage(e: CodeWhispererCodeScanException): String? {
+        val message = e.message
+        return when {
+            message == message("codewhisperer.codescan.invalid_source_zip_telemetry") -> {
+                message("codewhisperer.codescan.run_scan_error")
+            }
+            message.isNullOrBlank() -> null
```

Comment:
nit: Should this order be reversed, ie should isNullOrBlank be the the first condition?